wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -813,7 +813,7 @@ public void deleteProject(
   private static Map<String, String> parseAttrs(List<String> attributes)
       throws BadRequestException, ForbiddenException {
     if (attributes == null) {
-      return emptyMap();
+      return new HashMap<>();
```

Comment:
why this change?

Reply:
I believe it may be reverted after the last refactoring in this area. Will do

Reply:
reverted